core/src/main/scala/com/criteo/cuttle/WaitingTimesMonitor.scala

```diff
@@ -0,0 +1,18 @@
+package com.criteo.cuttle
+
+import scala.concurrent.duration._
+
+object WaitingTimesMonitor extends Queries {
```

Comment:
Also this code should be embeded into Executor. It's an implementation detail of the Executor.